### PR TITLE
Add a doc on how to write helm unitests.

### PR DESCRIPTION
**Pull request by danielhoherd**
Document how to write unittests so we can refer people to that when we ask them to write tests for their submissions, which has been a big turn off to people who are unfamiliar with testing helm charts.

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# Testing
+
+This project uses the [helm-unittest project](https://github.com/quintush/helm-unittest) to test the helm charts included within. Once that plugin is installed, you can run tests with `helm unitest -3 .`
```
Comment (kaxil) [inline tests/README.md:3]: We could also move to a more Pythonic way if helm-unittest needs to be forked  https://github.com/apache/airflow/blob/master/TESTING.rst#helm-unit-tests

WDYT? We use it for OSS Airflow and it has worked well for us so far, but I don't have a strong preference here.

Example commit to add a feature and add unit test for it in for OSS Airflow: https://github.com/apache/airflow/commit/441b4ef19f07d8c72cd38a8565804e56e63b543c#diff-f02a725253639ce892b023d3b6e307946244ebcba2b50af02feed402ca80ace9

Comment (danielhoherd) [reply tests/README.md:3]: Interesting. I had thought about the possibility of testing helm with python but never looked into it. I'll have to think about this some more.

Comment (jedcunningham) [reply tests/README.md:3]: I think this is a good idea. We should open another issue for it though.

File: tests/README.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+From this output, we can make the following assertions:
+
+```yaml
+---
+suite: Test templates/pgbouncer/pgbouncer-networkpolicy.yaml
+templates:
+  - templates/pgbouncer/pgbouncer-networkpolicy.yaml
+tests:
+  - it: "should work with pgbouncer.enabled: True, networkPolicies.enabled: True, pgbouncer.networkPolicies.enabled: True"
```
[the remark this answers is not in the document]
Comment (danielhoherd) [reply tests/README.md:62]: Yeah, good point. It's something we carried over from the original examples. I'm open to the idea of changing the wording.

**Review by kaxil — APPROVED** [review]

**Pull request merged**